### PR TITLE
koordlet: add more metrics for cpu throttled and node used memory

### PR DESCRIPTION
**Pull request by LeoLiuYan**
Signed-off-by: LeoLiuYan <929908264@qq.com>

### Ⅰ. Describe what this PR does

1. Add Prometheus metrics for node's and pod's cpu throttled ratio, including metrics below:

```
koordlet_pod_cpu_throttled_ratio{}
koordlet_container_cpu_throttled_ratio{}
```

2. Add Node memory used metrics, including metrics below:

```
koordlet_node_used_memory_bytes{}
```

### Ⅱ. Does this pull request fix one issue?

NONE

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

**Comment by jasonliu747** [comment]
@LeoLiuYan you could split this PR into two. One for adding metrics and the other for ConfigMap example.

**Comment by LeoLiuYan** [comment]
> @LeoLiuYan you could split this PR into two. One for adding metrics and the other for ConfigMap example.

OK.

File: pkg/koordlet/metrics/common.go [hunk 1]
```diff
@@ -44,11 +44,18 @@ var (
 		Help:      "Number of cpu cores used by node in realtime",
 	}, []string{NodeKey})
 
+	NodeUsedMemory = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: KoordletSubsystem,
+		Name:      "node_used_memory_bytes",
```
Comment (zwzhang0107) [inline pkg/koordlet/metrics/common.go:49]: I think these metrics have already exported by cadvisor, how about reuse this?
https://github.com/google/cadvisor/blob/master/docs/storage/prometheus.md

Comment (LeoLiuYan) [reply pkg/koordlet/metrics/common.go:49]: You are right. But should we  export the `node_used_memory_bytes`  metric?  @zwzhang0107

Comment (LeoLiuYan) [reply pkg/koordlet/metrics/common.go:49]: The way of koordlet calculates `node_used_memory` is: `memInfo.MemTotal - memInfo.MemAvailable`, `memInfo` is read from `/proc/meminfo`, it's different with `cadvisor`.  And the `node_used_memory` metric is used by `memory evict` feature.

That's a good idea to record `node_used_memory ` metric with `QoS` label. I will write code for this idea.  @zwzhang0107

**Pull request closed without merging**